pkg/apis/openebs.io/core/v1alpha1/csivolume.go

```diff
@@ -114,6 +115,20 @@ type ISCSIInfo struct {
 	Lun string `json:"lun"`
 }
 
+// CSIVolumeStatus status represents the current mount status of the volume
+type CSIVolumeStatus string
+
+// CSIVolumeStatusMounting indicated that a mount operation has been triggered
+// on the volume and is under progress
+const (
+	CSIVolumeStatusUninitialized      CSIVolumeStatus = ""
+	CSIVolumeStatusMountUnderProgress CSIVolumeStatus = "MountUnderProgress"
+	CSIVolumeStatusMounted            CSIVolumeStatus = "Mounted"
+	CSIVolumeStatusUnMounted          CSIVolumeStatus = "UnMounted"
+	CSIVolumeStatusRaw                CSIVolumeStatus = "Raw"
```

Comment:
others are self-explanatory.. comment would be helpful here.. and you can add for others also